packer/scripts/install-certstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -o errexit -o xtrace
+
+# Certstrap is used for creating the k8s signing keys, which are necessary to avoid communication bugs on host restarts
+echo "Installing certstrap ..."
+# We run chown in docker to avoid requiring sudo
+systemctl start docker.service
```

Comment:
I'm glad you brought this up... I think I should put a comment in here anyway to explain why the docker image isn't getting cleaned up right now.. and to help us remember to clean up if our vagrant provisioning ever stops using the same image.